modules/rke2/windows_worker/instances_worker.tf

```diff
@@ -13,7 +13,7 @@ resource "aws_instance" "windows_worker" {
   vpc_security_group_ids = ["${var.sg_id}"]
   key_name               = var.key_name
   tags = {
-    Name = "${var.resource_name}-windows-worker"
+    Name = "${var.resource_name}-distros-windows-worker"
```

Comment:
`resource_tag` here as well with `distros-qa` to be consistent?

Reply:
sure, forgot